### DISCUSSION TITLE
tauri build / bundle separation

### DISCUSSION THREAD

**Discussion by yk0n9** [tauri-apps/tauri#7291] [Q&A] [answered]
Can the build command of Tauri and the bundle of ```tauri.bundle.active``` be executed separately?
Because tauri build will compile the program and then bundle it, which is a seamless operation.
I need to do a lot of operations before the bundle, ```build.beforeBundleCommand``` doesn't seem to meet my needs.
Can any command or code be used to bundle the compiled program?

**Comment by FabianLars** [comment, marked as answer]
Not yet, no. We do like to make this possible in the future though. Until then there's only beforeBundleCommand
